src/core/server/saved_objects/routes/bulk_create.test.ts

```diff
@@ -17,52 +17,36 @@
  * under the License.
  */
 
-import Hapi from 'hapi';
-import { createMockServer } from './_mock_server';
-import { createBulkCreateRoute } from './bulk_create';
-// Disable lint errors for imports from src/core/* until SavedObjects migration is complete
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { savedObjectsClientMock } from '../../../../core/server/saved_objects/service/saved_objects_client.mock';
+import supertest from 'supertest';
+import { UnwrapPromise } from '@kbn/utility-types';
+import { registerBulkCreateRoute } from './bulk_create';
+import { savedObjectsClientMock } from '../../../../core/server/mocks';
+import { setupServer } from './test_utils';
+
+type setupServerReturn = UnwrapPromise<ReturnType<typeof setupServer>>;
 
 describe('POST /api/saved_objects/_bulk_create', () => {
-  let server: Hapi.Server;
-  const savedObjectsClient = savedObjectsClientMock.create();
+  let server: setupServerReturn['server'];
+  let httpSetup: setupServerReturn['httpSetup'];
+  let handlerContext: setupServerReturn['handlerContext'];
+  let savedObjectsClient: ReturnType<typeof savedObjectsClientMock.create>;
```

Comment:
So, these are mixed integration/unit test, as we are starting an http server and mocking the context provider. Tell me if I should move them in a `integration_tests` subfolder.

Reply:
I would move them. we should avoid affecting unit test time execution.